proto/frequenz/dispatch/v1/dispatch.proto

```diff
@@ -1,3 +1,5 @@
+// protolint:disable MAX_LINE_LENGTH
```

Comment:
Would it be a better option to tweak the documentation instead?

Reply:
IMO 80 character max line length enforcements are a relic of the 90s, so I would prefer to keep this. It was here before already, just further down.